### ISSUE TITLE
Initial Header is not reused

### ISSUE THREAD

**Issue by HexBlit** [emilsjolander/StickyListHeaders#321]
Hi I am trying to use these StickyHeaders with Fragments because I have a very complicated header system, in fact it is combined with Jeff Sharkey's multiple section implementation, using your library for the header of course. 
 http://jsharkey.org/blog/2008/08/18/separating-lists-with-headers-in-android-09/

The problem I have is that if i use the getHeaderView to do the Fragment commit it shows up as the initial header. but as soon as you start to scroll it wants try and get the HeaderView again, but doesn't see that the initial header you just had is the one you want to reuse so convertView is null and you end up recommiting the transaction and get the Parent already has child view error. 

In short, the library is not reusing the initial header view.
